### PR TITLE
Feature/phone verification

### PR DESCRIPTION
**Pull request by cglotr**
- Send phone verification code to clients via Twilio's Verify

### CODE REVIEW

File: v1.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+			b, err := json.Marshal(map[string]interface{}{
+				"api_key":      twilioAPIKey,
+				"country_code": int(countryCode),
+				"phone_number": int(phoneNumber),
+				"via":          "sms",
+			})
+
+			if err != nil {
+				log.Println(err)
```
Comment (imranariffin) [inline v1.go:69]: @cglotr Why don't we writeError here as well?
```suggestion
				log.Println(err)
				writeError(conn, xxxx)
```
Same goes to line 80, 88, 101. What do you think?

File: README.md [hunk 2]
```diff
@@ -9,4 +9,4 @@
 5. `go version`
 6. `go get github.com/hooligram/hooligram-server`
 7. `cd ~/go/src/github.com/hooligram/hooligram-server`
-8. `go run main.go&`
```
Comment (imranariffin) [inline README.md:12]: How about the step to define environment variables? Do you think we should include that? @cglotr

Comment (cglotr) [reply README.md:12]: Yeah, I agree. Done 21f9eddfa9a054a794b70b0410b13a354c6706c4

File: v1.go (hunk 1 above)
Comment (cglotr) [reply v1.go:69]: Yeah, I agree. But for *101*, the server will send a failure action `VERIFICATION_VERIFICATION_CODE_SENT_FAILED`

Comment (cglotr) [reply v1.go:69]: Done 704c212d13e3071005a081d2b48aca5a17fb1006

**Review by imranariffin — APPROVED** [review]
lgtm

**Pull request merged**